src/style/app.module.css

```diff
@@ -5963,17 +6785,17 @@ button[data-testid='createPostBtn'] {
 
 .createButtonEventHeader {
   background-color: var(--grey-bg-color);
-  color: var(--brown-color);
+  color: var(--dark-grey-font);
   width: 110px;
   /* margin-left: 5px; */
-  border: 1px solid var(--brown-color);
+  border: 1px solid var(--dark-grey-border);
   box-shadow: 2.5px 2.5px 2.5px rgba(0, 0, 0, 0.3);
 }
 
 .createButtonEventHeader:hover {
   background-color: var(--grey-bg-color);
   color: var(--black-color);
-  border: 1px solid var(--brown-color);
+  border: 1px solid var(--dark-grey-border);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`